.gitlab-ci.yml

```diff
@@ -268,22 +271,16 @@ wdl:
     - pwd
     - apt update && apt install -y default-jre
     - ${MAIN_PYTHON_PKG} -m virtualenv venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[all]
-    - make test tests=src/toil/test/wdl/wdltoil_test.py
+    - make marker="${MARKER}" test tests=src/toil/test/wdl/wdltoil_test.py
     - which java &> /dev/null || { echo >&2 "Java is not installed.  Install java to run these tests."; exit 1; }
-    - make test tests="src/toil/test/wdl/toilwdlTest.py src/toil/test/wdl/builtinTest.py"  # needs java (default-jre) to run "GATK.jar"
-    # Run some WDL conformance tests.
-    # These are not set up as Toil PyTest tests (yet?)
-    - git clone https://github.com/adamnovak/wdl-conformance-tests.git
-    - cd wdl-conformance-tests
-    - git checkout 0ca00774331630d8eb06990caa8a1ba178cabecb
-    - python run.py --runner toil-wdl-runner --versions 1.0 --threads 8
```

Comment:
Shouldn't cutting this be part of #4530 and not this PR?

I guess it's fine to do it here though.